src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -123,6 +128,11 @@ def __init__(
             default is ``True``. You can stop and start the streaming of the
             Fluent transcript subsequently via the method calls, ``transcript.start()``
             and ``transcript.stop()`` on the session object.
```

Comment:
```suggestion
            Fluent transcript subsequently with calls to the ``transcript.start()``
            and ``transcript.stop()`` methods on the session object.
```
Google dev stye guide recommends not using "via".

Reply:
Done. Thank you.